docs/concepts/topology.rst

```diff
@@ -0,0 +1,83 @@
+.. _topology_sdk:
+
+============
+QPU Topology 
+============
+
+To solve a :term:`bqm` on the D-Wave system, you
+must map it to a graph that represents the topology of the system's
+qubits. For D-Wave 2000Q systems, this is the *chimera* topology; for 
+next-generation Advantage systems, this is the *Pegasus* topology.
+
```

Comment:
same suggestion with the subsection links here.

Reply:
The headers in the RST file are rendered in the RTD theme as lines in the navigation bar:
![image](https://user-images.githubusercontent.com/34041130/76016452-b10fcd00-5ed1-11ea-9937-43e6b959fb1e.png)

If from another place we'll want to jump directly to a subsection here, we can either add a ref name above it then or use :doc:` <oceandocs:concepts/topology#subsection>`.

I'm not sure we'll want to jump to a subsection here and already now build generate hundreds of duplicate target warnings for the SDK because submodule repos have identical ref names.

Reply:
Oh I see. Thanks for letting me know. In that case, please ignore my suggestion.